frontend/packages/dev-console/integration-tests/features/topology/resources-Groupings-ODC-2922.feature

```diff
@@ -0,0 +1,94 @@
+Feature: Topology 
```

Comment:
Update feature description as "Resources List view in topology"

Reply:
Changed it to "Resources Groupings in Topology"